.travis.yml

```diff
@@ -14,5 +14,5 @@ script: yarn run validate && yarn run build
 deploy:
   provider: script
   skip_cleanup: true
-  script: bash _scripts/deploy.sh
+  script: npx semantic-release 
```

Comment:
Is it possible to leave off this line? Otherwise you're basically implemented #17. And in which case it should be:

```
npx travis-deploy-once "npx semantic-release"
```

per: https://github.com/semantic-release/semantic-release/blob/caribou/docs/recipes/travis.md#multiple-node-jobs-configuration

Reply:
I'll probably have to delete/comment out the entire block so travis won't error in the meantime, but yeah I can do that.

Reply:
Yeah if you feel confident that this approach works then I can do the `deploy` part in implementing #17